xla/debug_options_flags.cc

```diff
@@ -125,14 +125,6 @@ DebugOptions DefaultDebugOptionsIgnoringFlags() {
   opts.set_xla_gpu_enable_all_gather_combine_by_dim(true);
   opts.set_xla_gpu_enable_reduce_scatter_combine_by_dim(true);
 
-  opts.set_xla_gpu_enable_async_collectives(true);
-  opts.set_xla_gpu_enable_async_all_reduce(true);
-  opts.set_xla_gpu_enable_async_all_gather(false);
-  opts.set_xla_gpu_enable_async_collective_broadcast(true);
-  opts.set_xla_gpu_enable_async_collective_permute(false);
-  opts.set_xla_gpu_enable_async_all_to_all(false);
-  opts.set_xla_gpu_enable_async_reduce_scatter(false);
```

Comment:
Are these default values are not preserved? IIRC, enabling all async collectives by default caused issues in the past.

Reply:
The current behavior is that `xla_gpu_enable_async_collectives` overrides all other control knobs, that has been true by default for a while. So this pr wont change the current default settings.

Reply:
Thanks for clarifying